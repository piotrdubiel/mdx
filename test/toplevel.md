# Toplevel Reason

```ocaml env=main
# open Base
# let digit_alist =
    [ 0, "zero"; 1, "one"; 2, "two"  ; 3, "three"; 4, "four"
    ; 5, "five"; 6, "six"; 7, "seven"; 8, "eight"; 9, "nine" ]
val digit_alist : (int * string) list =
  [(0, "zero"); (1, "one"); (2, "two"); (3, "three"); (4, "four");
   (5, "five"); (6, "six"); (7, "seven"); (8, "eight"); (9, "nine")]
```