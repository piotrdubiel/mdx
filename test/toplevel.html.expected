<section id="toplevel-reason" class="level1">
<h1>Toplevel Reason</h1>
<div class="highlight">
<pre class="command-line" data-prompt="#" data-filter-output=">"><code class="language-reason">open Base;

let digit_alist = [
  (0, &quot;zero&quot;),
  (1, &quot;one&quot;),
  (2, &quot;two&quot;),
  (3, &quot;three&quot;),
  (4, &quot;four&quot;),
  (5, &quot;five&quot;),
  (6, &quot;six&quot;),
  (7, &quot;seven&quot;),
  (8, &quot;eight&quot;),
  (9, &quot;nine&quot;),
];

>val digit_alist : (int * string) list =
>  [(0, &quot;zero&quot;); (1, &quot;one&quot;); (2, &quot;two&quot;); (3, &quot;three&quot;); (4, &quot;four&quot;);
>   (5, &quot;five&quot;); (6, &quot;six&quot;); (7, &quot;seven&quot;); (8, &quot;eight&quot;); (9, &quot;nine&quot;)]
</code></pre>
</div>
</section>
